internal/jsre/deps/bindata.go

```diff
@@ -1,8 +1,7 @@
-// Code generated by go-bindata. DO NOT EDIT.
+// Package deps Code generated by go-bindata. (@generated) DO NOT EDIT.
```

Comment:
This is auto generated code, please don't fix these. We should report any issues upstream and have them fix it.

Reply:
Or is this the newly generated code?

Reply:
it is auto-generated code